presentation/src/main/java/me/androidbox/presentation/edit/screen/EditScreen.kt

```diff
@@ -52,9 +55,9 @@ fun EditScreen(
             .background(color = MaterialTheme.colorScheme.editScreenBackground)
         ) {
 
-            val fontSize =  when(titleType) {
-                TitleType.TITLE -> TitleType.TITLE.fontSize
-                TitleType.DESCRIPTION -> TitleType.DESCRIPTION.fontSize
+            val fontSize =  when(contentType) {
```

Comment:
This can all be replaced with:

val fontSize = contentType.fontSize

Since you already have the variable that has the content type, you can directly access fontSize and contentSizeRes

Reply:
@MKiperszmid Updated